src/app/core/admin/feedback/admin-list-feedback.component.ts

```diff
@@ -69,11 +153,48 @@ export class AdminListFeedbackComponent extends AbstractPageableDataComponent<an
 			});
 	}
 
+	getDefaultQuickFilters() {
+		return { open: { show: false, display: 'New/Open Issues' } };
+	}
+
+	getQuery(): any {
+		let query: any;
+
+		if (this.filters.open.show) {
+			query = {
+				$or: [{ status: FeedbackStatusOption.NEW }, { status: FeedbackStatusOption.OPEN }]
+			};
+		}
+		return query;
+	}
+
 	loadData(
 		pagingOptions: PagingOptions,
 		search: string,
 		query: any
-	): Observable<PagingResults<any>> {
+	): Observable<PagingResults<Feedback>> {
 		return this.feedbackService.getFeedback(pagingOptions, query, search, {});
 	}
+
+	updateFeedbackAssignee(index: number, assignee: string) {
+		this.feedbackService
+			.updateFeedbackAssignee(this.items[index].id, assignee)
+			.pipe(take(1), untilDestroyed(this))
+			.subscribe({
+				next: updatedFeedback => {
```

Comment:
@sshah-asymmetrik what happens if this call fails? Or the `updateFeedbackStatus` call below?